test/utils/widget_test_harness.dart

```diff
@@ -1,30 +1,35 @@
+import 'package:crowdleague/actions/redux_action.dart';
 import 'package:crowdleague/models/app/app_state.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_redux/flutter_redux.dart';
-import 'package:redux/redux.dart';
 
 import '../mocks/redux/redux_store_mocks.dart';
 
 /// A test harness to wrap a widget under test and provide all the functionality
 /// that a test may want in order to interact with the widget or check for
 /// expected values and behaviour.
 class WidgetTestHarness {
-  final Store<AppState> _store;
+  final FakeStore _fakeStore;
   final Widget _widgetUnderTest;
 
   WidgetTestHarness(
       {@required Widget widgetUnderTest,
-      Store<AppState> store,
+      FakeStore fakeStore,
       dynamic Function(AppStateBuilder) stateUpdates})
-      : _store = store ?? FakeStore(),
+      : _fakeStore = fakeStore ?? FakeStore(),
         _widgetUnderTest = widgetUnderTest {
     if (stateUpdates != null) {
-      _store.state.rebuild(stateUpdates);
+      _fakeStore.updateState(stateUpdates);
     }
   }
 
+  void updateAppState(dynamic Function(AppStateBuilder) updates) =>
```

Comment:
Allow the test to update the app state.  

I've added a style guide entry (mentioned in standup) to suggest that we should consider splitting up tests rather than changing the app state and continuing the test.  I put a comment on the function and left it in but we might consider taking it out at some point. 

[Style Guide > CONSIDER splitting up widget tests that change the app state](https://github.com/crowdleague/crowdleague/wiki/Style-Guide-(WIP)#consider-splitting-up-widget-tests-that-change-the-app-state)